.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: "🚀 publish"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    name: 🚀 NPM publish
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
```suggestion
        node-version: [16.x]
```

Reply:
Oh nvm, this repo uses 14 or greater... Isn't 14 out of support?

Ok, it is maintained  
https://endoflife.date/nodejs

Reply:
It'll be out of maintenance by mid of 2023, but it is a good idea to use 16.x instead.